pkg/store/kotsstore/kots_store.go

```diff
@@ -34,6 +35,8 @@ type KOTSStore struct {
 }
 
 func init() {
+	store.SetStore(StoreFromEnv())
```

Comment:
hmm, why the pattern change here? this makes it harder to add more implementations of the store interfaces in the future.